src/Services/ModulePermissionHandler.php

```diff
@@ -353,37 +66,46 @@ public function __construct(PermissionHandlerInterface $permission_handler, Modu
   }
 
   /**
-   *  Create the Site Builder role.
+   *  Create a role.
    */
-  public function createSiteBuilderRole() {
+  public function createRole($role_id, $role_name) {
     $role_storage = $this->entityTypeManager->getStorage('user_role');
-    // Check if the Site Builder role already exists.
-    $site_builder = $role_storage->load('site_builder');
-    if ($site_builder) {
+    // Check if the role already exists.
+    $content_editor = $role_storage->load($role_id);
```

Comment:
Generic role method should use generic variables. For future cleanup. Not going to mark it as blocking.

Reply:
Ah. Good catch! I'll change it.